### ISSUE TITLE
ios build error: SqflitePlugin.h

### ISSUE THREAD

**Issue by wilsonwg** [tekartik/sqflite#301]
I integrate sqlfite for my app. The android package was built successfully with error. But when I tried to build the ios package, Xcode gave the following error message:

'sqflite/SqflitePlugin.h' file not found

//  Generated file. Do not edit.
//

#import "GeneratedPluginRegistrant.h"
#import <sqflite/SqflitePlugin.h>     

@implementation GeneratedPluginRegistrant

+ (void)registerWithRegistry:(NSObject<FlutterPluginRegistry>*)registry {
  [SqflitePlugin registerWithRegistrar:[registry registrarForPlugin:@"SqflitePlugin"]];
}

@end

**Comment by alextekartik** [comment]
Have you tried all the tips in [troubleshooting](https://github.com/tekartik/sqflite/blob/master/sqflite/doc/troubleshooting.md)?

Does it happen with another plugin ? (`path_provider` is a good simple example)?

**Comment by wilsonwg** [comment]
Same thing happened to path_provider, and shared_preferences.

But after several hours of failure. I finally able to make the ios build and run on my device (iPad).
There are the things that I did:

1. Upgrade to flutter beta (refresh install, not upgrade).
2.  [Fix mac catalina related permision issue](https://github.com/flutter/flutter/issues/42302#issuecomment-539852516)
3. Open up the terminal, go the project root and type:
open ios/Runner.xcworkspace
In Xcode, select "Product > Run".

PS: 
1. I used to open Runner.xcodeproj in Xcode, but I guess now we should use Runner.xcworkspace
2. Must type open ios/Runner.xcworkspace for the first time, otherwise Xcode would grey out the run button.
